packages/sdk/src/errors/BrowserWalletNetworkMismatchError.ts

```diff
@@ -0,0 +1,6 @@
+export class BrowserWalletNetworkMismatchError extends Error {
+  constructor(message: string = "Network mismatch") {
```

Comment:
not required to assign a message in constructor by default since each wallet will indicate the message

Reply:
updated